idaes/models/properties/modular_properties/pure/tests/test_ChapmanEnskog.py

```diff
@@ -0,0 +1,166 @@
+
+from idaes.models.properties.modular_properties.pure.ChapmanEnskog import (
+    ChapmanEnskogLennardJones,
+    collision_integral_neufeld_callback,
+)
+from idaes.core.util.misc import add_object_reference
+from idaes.core.base.property_meta import PropertyClassMetadata
+
+
+def construct_dummy_model(param_dict):
+    m = ConcreteModel()
+
+    # Create a dummy parameter block
+    m.params = Block()
+
+    m.params.config = ConfigBlock(implicit=True)
+    # Parameters for sulfur dioxide, from Properties of Gases and Liquids 5th ed. Appendix B.
+    m.params.config.parameter_data = {
+        "lennard_jones_sigma": (param_dict["sigma"], pyunits.angstrom),
+        "lennard_jones_epsilon_reduced": (
+            param_dict["epsilon"],
+            pyunits.K,
+        ),
+    }
+
+    m.meta_object = PropertyClassMetadata()
+    m.meta_object._default_units.set_units(
+        temperature=pyunits.K,
+        mass=pyunits.kg,
+        length=pyunits.m,
+        time=pyunits.s,
+        amount=pyunits.mol,
+    )
+
+    def get_metadata(self):
+        return m.meta_object
+
+    m.get_metadata = types.MethodType(get_metadata, m)
+    m.params.get_metadata = types.MethodType(get_metadata, m.params)
+
+    # Create variables that should exist on param block
+    m.params.mw = Var(initialize=param_dict["mw"], units=pyunits.kg / pyunits.mol)
+    m.params.omega = Var(initialize=param_dict["omega"], units=pyunits.dimensionless)
+
+    # Create a dummy state block
+    m.props = Block([1])
+    add_object_reference(m.props[1], "params", m.params)
+
+    m.props[1].temperature = Var(initialize=273.16, units=pyunits.K)
+    return m
+
+
+@pytest.mark.unit
+def test_visc_vap_comp_sulfur_dioxide():
+    m = construct_dummy_model(
+        {
+            "sigma": 4.112,
+            "epsilon": 335.4,
+            "mw": 0.064065,
+            "omega": 0.257,
+        }
+    )
+
+    ChapmanEnskogLennardJones.visc_d_phase_comp.build_parameters(m.params, "Vap")
+
+    assert isinstance(m.params.lennard_jones_sigma, Var)
+    assert value(m.params.lennard_jones_sigma) == pytest.approx(4.112e-10, rel=1e-12)
+    assert isinstance(m.params.lennard_jones_epsilon_reduced, Var)
+    assert value(m.params.lennard_jones_epsilon_reduced) == 335.4
+    assert (
+        m.params.viscosity_collision_integral_callback
+        is collision_integral_neufeld_callback
+    )
+
+    expr = ChapmanEnskogLennardJones.visc_d_phase_comp.return_expression(
+        m.props[1], m.params, "Vap", m.props[1].temperature
+    )
+    expr_micropoise = pyunits.convert(expr, pyunits.micropoise)
+
+    # Pulled from Table 9.2, Properties of Gases and Liquids 5th Ed.
+    T_list = [10, 100, 300, 700]  # Gas temperature in C
+    visc_list = [120, 163, 246, 376]  # Experimental viscosities in millipoise
+    for i in range(4):
+        m.props[1].temperature.value = T_list[i] + 273.15
+        assert pyo.value(expr_micropoise) == pytest.approx(visc_list[i], rel=0.03)
```

Comment:
Minor comment: this a fairly loose tolerance (3%) and I see it used in other tests too, although I suspect that is unavoidable due to the uncertainty in the data. Just wanted to check to make sure these could not be tightened.

Reply:
We're testing against experimental values here, and that sort of tolerance is just what we get. The thermal conductivity methods are even worse, sometimes getting 15-30% error. At a certain point, getting the right order of magnitude is victory enough.